core/src/test/java/com/datastax/oss/driver/internal/core/config/typesafe/DefaultDriverConfigLoaderTest.java

```diff
@@ -294,4 +294,17 @@ public void should_load_setting_from_system_property_when_application_conf_is_al
       System.clearProperty("datastax-java-driver.basic.request.timeout");
     }
   }
+
+  @Test
+  public void should_create_from_string() {
+    DriverExecutionProfile config =
+        DriverConfigLoader.fromString("datastax-java-driver.basic.request.timeout = 1 millisecond")
```

Comment:
The string needs to be in the HOCON format, right? 
Are entries should be after newline (`/n`)?

Reply:
Yes. I should probably explain that in the javadocs.